azext_edge/edge/providers/check/base/pod.py

```diff
@@ -32,7 +32,7 @@ def evaluate_pod_health(
     target: str,
     pod: str,
     display_padding: int,
-    service_label: str,
+    service_label: str = None,
```

Comment:
Optional[str] ?

Reply:
Oh yeah probably, i just mirrored the type in the call to `get_namespaced_pods_by_prefix`, will update